classes/UpgradeSelfCheck.php

```diff
@@ -213,18 +213,25 @@ public function getRequirementWording(int $requirement, bool $isWebVersion = fal
             case self::PHP_COMPATIBILITY_INVALID:
                 return [
                     'message' => $this->translator->trans(
-                        'Your current PHP version isn\'t compatible with your PrestaShop version. (Expected: %s - %s | Current: %s)',
-                        [$phpCompatibilityRange['php_min_version'], $phpCompatibilityRange['php_max_version'], $phpCompatibilityRange['php_current_version']]
+                        'Your current PHP version isn\'t compatible with PrestaShop %s. (Expected: %s - %s | Current: %s)',
+                        [$version, $phpCompatibilityRange['php_min_version'], $phpCompatibilityRange['php_max_version'], $phpCompatibilityRange['php_current_version']]
                     ),
                 ];
+
             case self::ROOT_DIRECTORY_NOT_WRITABLE:
                 return [
-                    'message' => $this->translator->trans('Your store\'s root directory isn\'t writable. Provide write access to the user running PHP with appropriate permission & ownership.'),
+                    'message' => $this->translator->trans(
+                        'Your store\'s root directory %s isn\'t writable. Provide write access to the user running PHP with appropriate permission & ownership.',
+                        [$this->prodRootPath]
+                    ),
                 ];
 
             case self::ADMIN_UPGRADE_DIRECTORY_NOT_WRITABLE:
                 return [
-                    'message' => $this->translator->trans('The "/admin/autoupgrade" directory isn\'t writable. Provide write access to the user running PHP with appropriate permission & ownership.'),
+                    'message' => $this->translator->trans(
+                        'The %s directory isn\'t writable. Provide write access to the user running PHP with appropriate permission & ownership.',
+                        [$this->autoUpgradePath]
```

Comment:
Is the path displayed is sanitized? We should make sure the actual admin folder is not displayed to keep the merchants data safe in case he makes a screenshot of this page.

Reply:
Ok, so we should go with `/your-admin-directory/autoupgrade/` ?

Reply:
Yep, one good way would be to use the sanitizer currently present in the Logger class.

Reply:
I've added a TODO, I think we need to wait for a response to the wording comment in Figma before proceeding?